awx/plugins/inventory/openstack_inventory.py

```diff
@@ -57,15 +57,13 @@
 import sys
```

Comment:
I picked up this dependency update as I was doing the rest of the work.

The _inventory plugin_ needs `openstacksdk`, and I just had problems having that work with the old dependencies of the script before the rename, so I just gave up and upgraded it so they're both on the new libraries.

Reply:
`shade` is dead, long live `shade`

Reply:
I could swear I remember removing shade from requirements. Maybe it got added back in due to a rebase, and I gave up.

Do we still need shade? Can we make an issue to get rid of it?